homeassistant/components/light/nanoleaf_aurora.py

```diff
@@ -29,6 +29,9 @@
 
 CONFIG_FILE = '.nanoleaf_aurora.conf'
 
+NANOLEAF_AURORA_MIN_MIRED = 154
+NANOLEAF_AURORA_MAX_MIRED = 833
```

Comment:
The name of the const is a bit verbose and is used only once. In my opinion you could put the values direct to the methods.